DirectProgramming/C++SYCL_FPGA/Tutorials/Tools/experimental/platform_designer/README_2023-0.md

```diff
@@ -0,0 +1,330 @@
+# `Platform Designer` Sample
```

Comment:
As far as I can tell, most of my review comments from platform_designer/README.md also apply here.